src/main/java/de/dennisguse/opentracks/content/provider/ContentProviderUtils.java

```diff
@@ -351,10 +353,10 @@ public Marker createMarker(Cursor cursor) {
         return marker;
     }
 
-    public void deleteMarker(Context context, Marker.Id markerId) {
+    public int deleteMarker(Context context, Marker.Id markerId) {
```

Comment:
Why not return Marker.Id?

Reply:
It returns an integer (the number of rows deleted) because it's what delete in ContentResolver returns.